src/layers/legacy/medVtkInria/vtkImageView/vtkImageViewCornerAnnotation.cxx

```diff
@@ -29,59 +29,53 @@
 
 #include <cmath>
 
-#ifdef WIN32
-#define snprintf sprintf_s
-#endif
-
-vtkStandardNewMacro(vtkImageViewCornerAnnotation);
+vtkStandardNewMacro(vtkImageViewCornerAnnotation)
 
 vtkImageViewCornerAnnotation::vtkImageViewCornerAnnotation()
 {
     this->ImageView = nullptr;
+    this->view2d = nullptr;
 }
 
 vtkImageViewCornerAnnotation::~vtkImageViewCornerAnnotation() = default;
 
+void vtkImageViewCornerAnnotation::SetImageView(vtkImageView* arg)
+{
+    this->ImageView = arg;
+    this->view2d = vtkImageView2D::SafeDownCast (this->ImageView);
```

Comment:
this is done once for each data, not for each pixel as before

Reply:
Better!